src/List/List.jsx

```diff
@@ -0,0 +1,67 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PropTypes, PureComponent} from "react";
+
+import Item from "./Item";
+import MorePropTypes from "../utils/MorePropTypes";
+
+import "./List.less";
+
+const RowType = {
+  PLAIN: "plain",
+  BORDERED: "bordered",
+  ZEBRA: "zebra",
+};
+
+const cssClass = {
+  CONTAINER: "List",
+  EMPTY: "List--empty",
+  HEADER: "List--header",
```

Comment:
We should clarify in the docs then, my assumption was that all elements were PascalCased and only modifiers were camelCased.